runtime/src/main/java/org/corfudb/runtime/object/transactions/OptimisticTransactionalContext.java

```diff
@@ -102,7 +102,7 @@ public <R, T extends ICorfuSMR<T>> R access(ICorfuSMRProxyInternal<T> proxy,
                                     o::setUncommittedChanges);
```

Comment:
It won't let me comment on the actual line, but you need to change the parameter in:

```
long streamReadPosition = knownStreamsPosition.getOrDefault(proxy.getStreamID(), ts);
```

Reply:
Good catch. I have incorporated another of your comments and changed the key back into a UUID.